server/internal/config/viper.go

```diff
@@ -0,0 +1,39 @@
+// Copyright © 2023 OpenIMSDK open source community. All rights reserved.
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+func init() {
+	configPath := "./config.yaml"
```

Comment:
hard coded ~

Reply:
I need to specify a path to read the configuration file, do I need to consider reading the configuration file from the command line?